dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/handler/ExternalSourceEntryItemUriListHandler.java

```diff
@@ -23,21 +23,16 @@
 import org.dspace.authorize.AuthorizeException;
 import org.dspace.authorize.service.AuthorizeService;
 import org.dspace.content.Collection;
-import org.dspace.content.Item;
+import org.dspace.content.WorkspaceItem;
 import org.dspace.content.service.CollectionService;
 import org.dspace.core.Context;
 import org.dspace.external.model.ExternalDataObject;
 import org.dspace.external.service.ExternalDataService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.rest.webmvc.ResourceNotFoundException;
-import org.springframework.stereotype.Component;
 import org.springframework.web.bind.annotation.RequestMethod;
 
-/**
- * This class will handle ExternalSourceEntryUriList and it'll create Item objects based on them
- */
-@Component
-public class ExternalSourceEntryUriListHandler implements UriListHandler<Item> {
+public abstract class ExternalSourceEntryItemUriListHandler<T> implements UriListHandler<T> {
```

Comment:
Please add Javadoc description of this Abstract class

Reply:
This has been updated